timelock-server/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosTimeLockServer.java

```diff
@@ -88,6 +99,21 @@ private void registerPaxosResource() {
         environment.jersey().register(paxosResource);
     }
 
+    private void registerRateLimitingConfiguration(TimeLockServerConfiguration configuration) {
```

Comment:
This doesn't actually "register" the configuration (although it initialises it).

Let's return a `RateLimitingConfiguration` object, set it in `onStartup`, and call this method `getRateLimitingConfiguration`.